third-party/realdds/include/realdds/dds-defines.h

```diff
@@ -36,4 +37,39 @@ using dds_entity_id = eprosima::fastrtps::rtps::EntityId_t;
 typedef int dds_domain_id;
 
 
+// Internal properties of a video stream.
+// The sensor has a lens and thus has properties such as a focal point, distortion, and principal point.
+struct video_intrinsics
+{
+    int   width = 0;                    // Horizontal resolution, in pixels
+    int   height = 0;                   // Vertical resolution, in pixels
+    float principal_point_x = 0.0f;     // Pixel offset from the left edge
+    float principal_point_y = 0.0f;     // Pixel offset from the top edge
+    float focal_lenght_x = 0.0f;        // As a multiple of pixel width
+    float focal_lenght_y = 0.0f;        // As a multiple of pixel height
+    int   distortion_model = 0;         // Distortion model of the image
+    float distortion_coeffs[5] = { 0 }; // Distortion model coefficients
+
+    bool is_valid() const { return focal_lenght_x > 0 && focal_lenght_y > 0; }
+    bool operator<( const video_intrinsics & rhs ) const {
```

Comment:
This is needed as a demand of `std::set`?

Reply:
Yes, set checks equality as !(a<b) && !(b<a)